src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -177,6 +177,69 @@ export const SueReportLocation = ({
     return <LoadingSpinner loading />;
   }
 
+  const onMapPress = async ({
+    nativeEvent
+  }: {
+    nativeEvent: { action: string; coordinate: Location.LocationObjectCoords };
+  }) => {
+    if (nativeEvent.action !== 'marker-press' && nativeEvent.action !== 'callout-inside-press') {
+      setUpdatedRegion(false);
+      setUpdateRegionFromImage(false);
+      setSelectedPosition(nativeEvent.coordinate);
```

Comment:
we could call this always first where used, maybe this helps a bit for "speed of visual feedback" when pressing the map? it shouldn't hurt to move it up, right?

Reply:
Are you talking about `setSelectedPosition`?

Reply:
yes

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1140/commits/1f80a77877fa798a7f4fc8c4d88ef7cbd3af53ef